docs/experiments-nfcorpus.md

```diff
@@ -133,7 +134,7 @@ python -m pyserini.search.faiss \
   --index indexes/nfcorpus.bge-base-en-v1.5 \
   --topics collections/nfcorpus/queries.tsv \
   --output runs/run.beir.bge-base-en-v1.5.nfcorpus.txt \
-  --batch 128 --threads 32 \
+  --batch 128 --threads 8 \
```

Comment:
I think that thread 32 is too big for personal computers. Performance improves from ~10it/s to ~40it/s on my machine.